test/e2e/fixture/fixture.go

```diff
@@ -136,22 +133,15 @@ func EnsureCleanState() {
 
 	start := time.Now()
 
+	policy := v1.DeletePropagationBackground
 	// delete resources
-	text, err := Run("", "kubectl", "get", "app", "-o", "name")
-	CheckError(err)
-	for _, name := range strings.Split(text, "\n") {
-		if name != "" {
-			appName := strings.TrimPrefix(name, "application.argoproj.io/")
-			// we cannot delete any app, if an op is in progress
-			_, _ = RunCli("app", "terminate-op", appName)
```

Comment:
Do you need to maintain this line?

Reply:
There is no need to terminate operation after fixing https://github.com/argoproj/argo-cd/issues/1665.  After application is deleted the controller is no longs stuck trying to update operation state.

Reply:
👍